compiler/core/lam_convert.ml

```diff
@@ -253,6 +253,7 @@ let lam_prim ~primitive:(p : Lambda.primitive) ~args loc : Lam.t =
   | Pandint -> prim ~primitive:Pandint ~args loc
   | Porint -> prim ~primitive:Porint ~args loc
   | Pxorint -> prim ~primitive:Pxorint ~args loc
+  | P_bitnot_int -> prim ~primitive:P_bitnot_int ~args loc
```

Comment:
Looking at `Pandint`, `Porint`, etc. - shouldn't it be `Pbitnotint` for consistency (even though I agree that naming convention is a bit weird)?

Reply:
Internal names of primitives are mess, I will rename all of them anyway after the major release